### PR TITLE
PR-files-rootdir-bug

### PR DESCRIPTION
**Pull request by mainmode1**
- [X] `npm run test` succeeds.
- [X] `npm run lint` succeeds.
- [ ] Appropriate changes to README are included in PR.

### Changes
- **`files`**
  - Fxed file name mangling bug when `rootDir = './'`  in `getProjectFiles()` introduced by [9297b98](https://github.com/google/clasp/commit/9297b98df6c2c11033319b4e440a1fef0ebc84ee)

### CODE REVIEW

File: src/files.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export function getProjectFiles(rootDir: string = path.join('.', '/'), callback:
           // Preserves subdirectory names in rootDir
           // (rootDir/foo/Code.js becomes foo/Code.js)
           let formattedName = nameWithoutExt;
-          if (rootDir) {
+          if (rootDir && rootDir !== './' /* Extra check just in case so filename not mangled */) {
```
Comment (grant) [inline src/files.ts]: Is this check really necessary?

**Comment by grant** [comment]
Conflicts.

File: src/files.ts (hunk 1 above)
Comment (mainmode1) [reply src/files.ts]: Not any more 🙂. Bug introduced by [PR #292](https://github.com/google/clasp/pull/292) ([9297b98](https://github.com/google/clasp/commit/9297b98df6c2c11033319b4e440a1fef0ebc84ee)). I think this is now superseded by [PR #323](https://github.com/google/clasp/pull/323).

**Comment by mainmode1** [comment]
@grant Superseded by [PR #323](https://github.com/google/clasp/pull/323). Clan be closed.

**Pull request closed without merging**